src/main/java/org/mule/extension/oauth2/api/authorizationcode/DefaultAuthorizationCodeGrantType.java

```diff
@@ -151,6 +156,33 @@
   @Optional(defaultValue = "true")
   private boolean encodeClientCredentialsInBody;
 
+  @Override
+  public boolean equals(Object obj) {
+    if (obj instanceof DefaultAuthorizationCodeGrantType) {
```

Comment:
Doesn't this have a super class with more stuff to consider?

Reply:
good catch